### DISCUSSION TITLE
Is there a way to reset cache for unmatched items?

### DISCUSSION THREAD

**Discussion by Floydian-dk** [Taxel/PlexTraktSync#1160] [Q&A] [answered]
Have some movies & TV-shows which wasn't on Trakt when I started using PlexTraktSync.
I've since created some of them in TMDB and then imported them into Trakt, however PlexTraktSync didn't import my watch history of these movies/shows.
Tried to remove the cache database, and by doing that the watch history was synced.

Have I missed a setting in my configuration or would it be possible to make a setting that allow PlexTraktSync to try to re-match unmatched movies/episodes without deleting the entire cache?

**Comment by simonc56** [comment, marked as answer]
Maybe using Expiration setting of [requests-cache](https://github.com/requests-cache/requests-cache).
- https://requests-cache.readthedocs.io/en/stable/user_guide/expiration.html

**Reply by Floydian-dk** [reply]
Thank you for the suggestion. Will have a look at it, and see if I can figure out how to make the necessary changes.

**Reply by simonc56** [reply]
may be solved by #1221

**Reply by Floydian-dk** [reply]
Cool... I'll test it out :-)
Thank you for working on this - it's highly appreciated.
